app/src/main/res/layout/activity_setting.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
+    android:background="@drawable/popup_background"
+    tools:context=".Setting">
+
+    <ImageButton
+        android:id="@+id/setting_exit"
+        android:layout_width="30dp"
+        android:layout_height="30dp"
+        android:layout_margin="30dp"
+        android:layout_gravity="end"
+        android:onClick="exitPage"
+        android:background="@drawable/close"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentRight="true"/>
+
+    <TextView
+        android:id="@+id/textView_settings"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textColor="@color/black"
+        android:text="Settings"

```

Comment:
Extract string resource here. Click on the line, and use `Alt + Enter` to see a list of suggested actions, one of which is "Extract String Resource"